cilium-dbg/cmd/encrypt_status.go

```diff
@@ -92,6 +93,9 @@ func dumpIPsecStatus() (models.EncryptionStatus, error) {
 	if err != nil {
 		return models.EncryptionStatus{}, fmt.Errorf("error getting IPsec decryption interfaces: %w", err)
 	}
+	if option.Config.TunnelingEnabled() && len(decryptInts) == 0 {
```

Comment:
@pchaigno correct me if I'm wrong, but in the original issue, I think the suggestion was to actually find where we attach our eBPF program too that does the decap processing? This solution looks a bit too simple.